ci-operator/config/openshift/multiarch/openshift-multiarch-master__nightly-4.14.yaml

```diff
@@ -700,9 +700,11 @@ tests:
       TEST_SUITE: upgrade-conformance
     workflow: openshift-upgrade-gcp-heterogeneous
 - as: ocp-e2e-sno-power
-  cron: 0 4 * * *
+  cron: 0 4 * * 3
```

Comment:
Do we have a 4.15 version of this job as well? I noted we updated the timing of 4.14 and master, but it seems like maybe we need to stagger across all versions?

Reply:
no, do we need it for 4.15?

Reply:
will do it.